### ISSUE TITLE
it do not work  on hhvm

### ISSUE THREAD

**Issue by Legacy-Launcher** [budgetneon/v2pagecache#1]
Component   Detected            Status
PHP             5.6.99-hhvm supported (PHP 5.4 or greater)
SAPI                    srv                 unsupported (pagecache has not been tested with this SAPI)

how can i modify it to work?

**Comment by budgetneon** [comment]
Sorry, I have no idea.   I am familiar with hhvm, but have never run it.

The code just looks at the sapi name, using php_sapi_name().  Hhvm returns "srv".   That's not in the list of things we've tested, like "cgi-fgci", "apache2handler", and "litespeed".

It's possible that it does work, the language you're seeing just means we haven't ever tested it.

Have you tried it?  What happens?

**Comment by budgetneon** [comment]
Also, I wasn't aware opencart itself was working with hhvm yet.   Are you successfully running opencart in hhvm without issues?

**Comment by keramist** [comment]
opencart 2.0.3.1 works on hhvm but 1.5.6.4 not
aslo v2pagecache works on nginx but not work on nginx+hhvm

**Comment by budgetneon** [comment]
Okay.  I'll leave the issue open, but we don't currently have an hhvm environment to test with, so we won't be working on it any time soon.

**Comment by keramist** [comment]
you can use these script to install hhvm for test
centos 6.5 /6.6+nginx+hhvm fast and easy install
https://github.com/lj2007331/lnmp

**Comment by budgetneon** [comment]
Right, I understand we could get another environment and install, but our environments are already in use.
